src/codeu/chat/server/Server.java

```diff
@@ -63,6 +57,11 @@
 
   private final Relay relay;
   private Uuid lastSeen = Uuid.NULL;
+  /* This was added to create a ServerInfo object*/
```

Comment:
Nit: Usually in big projects with big team member, no one will follow when this added. Generally you have to document what is this variable, and in what way it used if it's not obvious.